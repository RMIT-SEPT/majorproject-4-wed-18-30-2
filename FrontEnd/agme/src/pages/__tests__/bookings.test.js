import React from "react";
import { mount } from "enzyme";
import ReactDOM from 'react-dom';
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import renderer from 'react-test-renderer';


//? Pages
import Bookings from "../bookings";
import {Link} from "react-router-dom";


it('renders correctly', () => {
    const tree = renderer
        .create(<Bookings page="/bookings">Bookings</Bookings>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});




Enzyme.configure({ adapter: new Adapter() });

describe('Booking page test', () =>{
    //? 1. Check if Sign-up Page Render Properly
    test('renders learn react link', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Bookings />, div);
    });
;

});