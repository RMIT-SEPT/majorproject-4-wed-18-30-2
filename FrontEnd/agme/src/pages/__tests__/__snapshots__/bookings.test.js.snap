// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="container"
>
  <h1
    id="title"
  >
    List of Bookings
  </h1>
  <table
    id="datas"
  >
    <tbody
      id="tblData"
    >
      <tr>
        <th>
          ID
        </th>
        <th>
          Service
        </th>
        <th>
          Worker
        </th>
        <th>
          Date & Time
        </th>
      </tr>
    </tbody>
  </table>
  <div>
    <form
      className="login-form"
      onSubmit={[Function]}
    >
      <h1
        className="font-weight-bold"
        id="heading"
      >
        Delete Booking
      </h1>
      <div
        className="alert alert-danger d-none"
        id="errorMessage"
      />
      <div
        className="alert alert-success d-none"
        id="successMessage"
      />
      <div
        className="form-group"
      >
        <label
          className=""
        >
          Booking ID
        </label>
        <input
          className="form-control"
          id="booking-id"
          name="booking-id"
          placeholder="Booking ID"
          type="text"
        />
      </div>
      <button
        aria-label={null}
        className="btn-lg btn-success btn-block mt-4 mb-3 btn btn-secondary"
        onClick={[Function]}
      >
        Delete
      </button>
    </form>
  </div>
</div>
`;
